packages/components/molecules/f-tabs/src/index.js

```diff
@@ -6,13 +6,15 @@
 
 
 // Import vue component
-import VueTabs from '@/components/Tabs.vue';
+import Tabs from '@/components/Tabs.vue';
+import Tab from '@/components/Tab.vue';
```

Comment:
Guessing there's a reason, but what's the reason to export both components, rather than pull in the `Tab` component into the `Tabs.vue`?

Reply:
Well the thought process was that technically you cou make your own version of `Tab.vue` if you needed a really custom tab for some reason. As you can register to the `Tabs.vue` by injecting the provided callback

Reply:
Ok, cool. As long as we have it documented why we've done that in the README, then I'm happy.

Reply:
I have added a bit into the readme to explain